test/test_score.js

```diff
@@ -61,5 +62,18 @@ describe('calculFrame', function(){
 
   );
  }
- 
- );
\ No newline at end of file
+ );
+
+describe('calcFrame10', function() {
+  it('should return the correct result of frame 10', function() {
```

Comment:
It could be better to create a desc for each tests

 it('test_description', function() {
assert.equal(calcFrame10(10, 5, 5),20);
});